tests/testsuite/init.rs

```diff
@@ -1,16 +1,11 @@
 //! Tests for the `cargo init` command.
 
-use cargo_test_support::{command_is_available, paths, Execs};
-use std::env;
+use cargo_test_support::compare::assert;
```

Comment:
> Something observations when dealing with the migration:
>
> - snapbox hasn't yet support unordered assertion.

Was this needed for this migration?  If so, I missed it and want to look closer.

I was planning on creating issues to map out the migration which included unordered.  You going ahead is valuable though for being able to have this conversation vs just having help-wanted issues where we might not get as much feedback due to people assuming everything they come across is expected.

Reply:
Not really. At least not in this PR. I deliberately picked `cargo-init` because it doesn't contain unordered ones. `cargo-build` and other build commands should have lots of unordered assertions.